### PR TITLE
Committee

### PR DESCRIPTION
**Pull request by gunmack**
- Created an initial format for the committees page
- Just need pictures and info

image

### CODE REVIEW

**Comment by micahdbak** [comment]
So, @gunmack , this PR is a bit more topical now that the old common room is kil.

Maybe rework the page to just talk about the new common room? You can just use the existing photo of the room and we can upgrade it later when the room is better set up.

If you wanna be cool, feel free to add a "RIP" section in the bottom of the page in memoriam of the old common room.

Also prices for snacks and drinks will be the same so keep those tables - the machines will be in the new room!

**Comment by micahdbak** [comment]
(wrong pr ignore that comment)

**Comment by micahdbak** [comment]
@gunmack - wanna keep working on this or no?

Take a look in the issue for text on each committee, + each committee

**Comment by gunmack** [comment]
> @gunmack - wanna keep working on this or no?
> 
> 
> 
> Take a look in the issue for text on each committee, + each committee

I'd like to keep working on it

**Comment by gunmack** [comment]
@micahdbak Thoughts?
image
image

**Comment by micahdbak** [comment]
oop forgot to reply to this, but just curious, what happens when you hover over the name of one of the committees? does it light up or otherwise indicate that you can click on it to display committee information?

**Comment by gunmack** [comment]
@micahdbak 

https://github.com/user-attachments/assets/8e23bb1c-56b5-483c-8308-b26a90f7b207

https://github.com/user-attachments/assets/e8b05425-1622-4970-8f02-ec5bb69882fd

**Comment by micahdbak** [comment]
omg i love it, tysm <3

File: src/main/pages/Committees.js [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+                  toggle={() => toggleSection(committee.name)}
+                />
+              );
+            }
+          })}
+        </div>
+
+        <div>
+          {committees2.map((committee) => {
+            if (committee.key === 'Policy') {
```
Comment (micahdbak) [inline src/main/pages/Committees.js:283]: You could move this if to the above map; like so:

```js
committees2.map((committee) => {
  if (committee.key === 'W3C') {
    return /* ... */;
  } else if (committee.key === 'Policy') {
    return /* ... */;
  }
});
```

**Review by micahdbak — APPROVED** [review]
One minor change; and if you could merge in the main branch that'd be great!

Aside from that, consider this PR approved

**Comment by micahdbak** [comment]
🥳 ty Julkies

**Pull request merged**